### DISCUSSION TITLE
How to hide title bar for progressive web apps (PWA)

### DISCUSSION THREAD

**Discussion by cristianrgreco** [i3/i3#5497] [Q&A]
Hello,

PWAs of Chromium based browsers seem to be adding their own title bar. See screenshot where GitHub is running as its own app:

![Screenshot_20230430_114636](https://user-images.githubusercontent.com/10348798/235348999-02349817-89d5-4630-9229-c8eb0db2b319.png)

It seems this is part of the web app itself, so I don't know if it can be managed by i3. 
I am already setting default window borders, as shown in alacritty on the right.

Any help appreciated. Thanks!

**Comment by arakis** [comment]
I would like to have the opposite - showing the title bar. The reason is, so i can see the focus. But in my case, PWA does not have a title bar :-/

EDIT:

after research, I found the solution:
```
for_window [class="Google-chrome"] border normal
```

In your case, you could use:
```
for_window [class="chromium"] border none
```

But then all chromium windows won't show a border. To find out the alternative, more specific class, execute `xprop` in a terminal and click on the PWA. You will see something like:
```
WM_CLASS(STRING) = "crx_hnpfjngllnobngcgfapefoaidbinmjnm", "chromium"
```

Take the more specific class.

EDIT 2:

Sorry, I didn't checked your screenshot properly. The border you're talking about is a custom border of chrom[e|ium] itself! It's not an i3 border. So this is a chrom[e|ium] issue.
